chrome/browser/resources/record_replay/record_replay.ts

```diff
@@ -14,6 +14,15 @@
 // we should also be able to add mojo bindings or use `chrome.send()`
 // and other things mentioned in webui_explainer.md.
 
-import "./auth.js";
+//import "./auth.js";
```

Comment:
I disabled these imports because the JS was throwing an exception that was suppressing my later messages from showing up.